### PR TITLE
Rec form

### LINKED ISSUE #6: User Story 4 - Create Playlist from Dashboard
As a registered user,
when I'm on the user profile, '/profile', 
I see a button to "Create a New Playlist". 

I'm taken to a new page, '/create_playlist',  where I can input the cuisine I'm cooking and mood. 

These options will be in a dropdown menu. I can select an option in each dropdown menu and then click on "Submit".  

- [ ] Build button "Create Playlist"
- [ ] Create Cuisines Dropdown: Italian, Mexican, Chinese, Japanese, Indian, Korean, American, Thai, Seafood, Mediterranean, Pizza, French, Spanish, Greek, BBQ. 
- [ ] Create Mood Dropdown: Romantic, Cocktail Party, Chill, Dinner Party, Dance Party, Backyard Hits, Day Drinking, Fancy, Holiday, Dramatic, Depressing/Glum, Rainy Day.

### PR DESCRIPTION
**Pull request by linda-le1**
**Functionality:**
Add drop down choices for cuisine and mood for recommendations. 

**Type of Change:**
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which maintains existing functionality)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

**Testing**(RSpec tests and development environment):
100% in RSPEC

**Related Issues:**
Closes #6

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.3'
-
+gem "bootstrap_form"
```
Comment (mel-rob) [inline Gemfile]: Glad you found this!

File: app/views/recommendations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<%= bootstrap_form_tag url: '/recommendations' do |f| %>
+<%= f.select :cuisine, [["Italian", "italian"], ["Mexican", "mexican"], ["Indian", "indian"], ["American", "american"], ["Thai", "thai"], ["Greek", "greek"], ["Chinese", "chinese"], ["Japanese", "japanese"], ["Latin", "latin"], ["Vietnamese", "vietnamese"], ["BBQ", "bbq"], ["Korean", "korean"], ["French", "french"]],
```
Comment (mel-rob) [inline app/views/recommendations/new.html.erb]: @linda-le1 @jordanholtkamp - To confirm, the first "Italian" is what the user sees in the drop down, and the second "italian" is what we are going to pass in via params to beet farmer? If so, we'll actually need to adjust the params based on the endpoint mapping. For example, we want the "Italian" selection to actually use a param of "ital" for the endpoint based on research. There is an endpoint mapping markdown file in beet farmer master for reference. I'm also happy to make these changes if that's easier -- just lmk if that's how it works!

Comment (mel-rob) [reply app/views/recommendations/new.html.erb]: Here is the endpoint mapping doc: https://github.com/linda-le1/beet-farmer/blob/master/endpoint_mappings.md

**Comment by jordanholtkamp** [comment]
Will work out what is being passed over to sinatra later

**Pull request merged**